django-backend/fecfiler/user/models.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+import uuid
+from django.utils.translation import gettext_lazy as _
+
+
+class User(AbstractUser):
+    id = models.UUIDField(
+        default=uuid.uuid4,
+        editable=False,
+        primary_key=True,
+        serialize=False,
+        unique=True,
+    )
+    first_name = models.CharField(
+        _("first name"), max_length=150, null=True, blank=True
```

Comment:
small nit: we probably don't need _("first name") and _("last name")

Reply:
agreed.  pulled it from django source because we wanted to add `null=True`